### PR TITLE
Add dsnparse feedstock

### PR DESCRIPTION
**Pull request by Juanlu001**
Created directly with `conda skeleton` and minimum editing.

### CODE REVIEW

File: recipes/dsnparse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+test:
+  imports:
+    - dsnparse
+
+about:
+  home: http://github.com/Jaymon/dsnparse
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/dsnparse/meta.yaml]: Please, add the content of the ``license_file``

Comment (Juanlu001) [reply recipes/dsnparse/meta.yaml]: Thanks, I had missed that! Fixed

**Comment by Juanlu001** [comment]
I tried patching `MANIFEST.in` to include the license file but apparently it's not enough. Before I send a pull request upstream, can someone confirm that this is the right way to go?

**Comment by synapticarbors** [comment]
The `MANIFEST.in` determines what goes in the tarball (the sdist) that goes on pypi. Patching will have no effect since by the time you fetch the tarball it already hasn't been included. You should put in a pull request in the upstream repo and just manually package the `license_file` here:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/dsnparse/0001-Include-License-in-source-distribution.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
+From 7ab57006a38816c13a9ef37a5df7aad63b136459 Mon Sep 17 00:00:00 2001
```
Comment (synapticarbors) [inline recipes/dsnparse/0001-Include-License-in-source-distribution.patch]: Delete this patch and manually package the license file:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/dsnparse/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6252d2a0b12e7c5bf1775e77ed0d3ee0a06f92937cdb2d2cac6ee039b557d0b4
+  patches:
+    - 0001-Include-License-in-source-distribution.patch
```
Comment (synapticarbors) [inline recipes/dsnparse/meta.yaml]: remove the patch.

File: recipes/dsnparse/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/dsnparse/meta.yaml]: It looks like this could be `noarch: python`

https://conda-forge.org/docs/meta.html#building-noarch-packages

**Comment by Juanlu001** [comment]
Thanks @synapticarbors! Tracking here: https://github.com/Jaymon/dsnparse/pull/5

**Comment by Juanlu001** [comment]
@conda-forge-admin, please add noarch: python

**Comment by Juanlu001** [comment]
Error seems unrelated, will retry tomorrow

**Comment by synapticarbors** [comment]
Just a note, I don't think you can request noarch from the webservices from staged-recipes. I think it only works from a feedstock. I've just added it manually myself.

**Comment by Juanlu001** [comment]
From the docs you sent :smiley: 

> or you can just ask the webservice to add it for you and rerender: say @conda-forge-admin, please add noarch: python in an open PR.

Anyway, thanks! It's still failing with the same error though

**Comment by Juanlu001** [comment]
There's an upstream issue: https://github.com/conda/conda-build/issues/3114

**Comment by drnextgis** [comment]
It looks like an upstream issue was fixed.

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge``!  :)

**Pull request merged**